src/NodaTime.Test/TimeZones/BclDateTimeZoneTest.cs

```diff
@@ -110,8 +132,12 @@ private void ValidateZoneEveryWeek(TimeZoneInfo windowsZone)
         {
             var nodaZone = BclDateTimeZone.FromTimeZoneInfo(windowsZone);
 
+            // Currently .NET Core doesn't expose the information we need to determine any DST recurrence
+            // after the final tzif rule. (Bug to be filed against corefx.) For the moment, limit how far we check.
```

Comment:
Ditto. Also is this worth extracting to a test helper anywhere?

Reply:
I would do if we needed it more than twice, or permanently. As I'm hoping to get rid of this, it didn't feel worth it.